frontend/src/pages/Lobbies.tsx

```diff
@@ -1,30 +1,110 @@
 import '../App.css';
-import React from 'react';
 import Header from '../components/header';
+import AllLobbies from '../components/lobbies.json'
+import CreateLobbyForm from '../components/createlobbyform'
+import React, {useEffect, useState } from 'react';
+import { fetchGamesServiceAPI, Game, GamesServiceResponse, getGameImageUrl} from '../utilities';
+import { useAuth } from '../context/AuthContext';
+
+export default function Lobbies() {
+  // State for the games sidebar
+  const {getUser, getAuthToken, setupUser} = useAuth();
+  const [gameResults, setGameResults] = useState<Game[]>([]);
+
+  // Fetches current users games, updates if user changes 
+  useEffect(() => {
+    const fetchGames = async () => {
+      try {
+        const gamesServiceData: GamesServiceResponse | null = await fetchGamesServiceAPI(getAuthToken());
+
+        if (gamesServiceData !== null) {
+          // Update games stored in the User
+          setupUser(JSON.stringify({
+            ...getUser(),
+            games: gamesServiceData.list_of_games.games
+          }));
+        }
+
+        const gameResults: Game[] = getUser().games;
+        setGameResults(gameResults)
+      } catch (err) {
+        console.error(err);
+      }
+    };
+
+    fetchGames(); 
+  }, [getAuthToken, getUser, setupUser]);
+
+   // State for creating lobby 
+   const [showCreateForm, setShowCreateForm] = useState(false);
+
+   const handleCreateLobby = () => {
+    setShowCreateForm(!showCreateForm);
+   };
 
-function Lobbies() {
   return (
     <div className="overflow-hidden h-screen">
       <Header></Header>
       <div className="flex flex-row h-screen text-white">
-        <div className="bg-[#4C4C4C] w-1/6 text-xl font-bold">
+        <div className="bg-[#4C4C4C] w-1/4 text-xl font-bold  overflow-y-auto"> 
           <p className="pt-2 text-center">
-          Games
+            Games
           </p>
+          <hr className="h-px mb-8 mt-4 bg-white border-0 dark:bg-gray-500"/>
+          <ul className=''>
+            {gameResults.map(game => 
```

Comment:
Since you will use getUser to retrieve the games, you can probably remove the gameResults state as you won't need it anymore